Dockerfile

```diff
@@ -1,12 +1,15 @@
-FROM golang:1-alpine as builder
+FROM golang:1.15-alpine as builder
 
-RUN apk add --no-cache git pkgconfig ffmpeg-dev build-base
+RUN apk add --no-cache make gcc musl-dev linux-headers git
+
+RUN apk add --no-cache git pkgconfig gnutls-dev ffmpeg-dev build-base
 
 WORKDIR /root
 RUN wget https://storage.googleapis.com/lp_testharness_assets/official_test_source_2s_keys_24pfs.mp4
 RUN wget https://storage.googleapis.com/lp_testharness_assets/official_test_source_2s_keys_24pfs_3min.mp4
 RUN wget https://storage.googleapis.com/lp_testharness_assets/bbb_sunflower_1080p_30fps_normal_t02.mp4
 RUN wget https://storage.googleapis.com/lp_testharness_assets/bbb_sunflower_1080p_30fps_normal_2min.mp4
+RUN wget https://storage.googleapis.com/video.livepool.io/official_test_source_2s_keys_24pfs_30s.mp4
```

Comment:
Can this video file be put into the same storage directory as the other video files referenced in this Dockerfile?

Reply:
Sure, I requested access in discord

Reply:
fixed in 5c4148d